include/net/net_if.h

```diff
@@ -271,6 +274,28 @@ struct net_if {
 	} ipv4;
 #endif /* CONFIG_NET_IPV4 */
 
+#if defined(CONFIG_NET_IPV4_AUTOCONF)
+    struct {
+        /** Number of sent probe requests */
+        uint8_t probe_cnt;
+        /** Number of sent announcements */
+        uint8_t announce_cnt;
+        /** Incoming conflict count */
+        uint8_t conflict_cnt;
+        /** Requested IP addr */
```

Comment:
2 times the same comment on requested IP addr. 

Btw, couldn't current_ip be removed and you would use one of the address slot in ipv4 struct above?